frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/InteractiveEmbeddingCTA/test/enterprise.unit.spec.tsx

```diff
@@ -25,7 +25,7 @@ describe("InteractiveEmbeddingCTA", () => {
 
     expect(screen.getByTestId("interactive-embedding-cta")).toHaveAttribute(
       "href",
-      "https://www.metabase.com/product/embedded-analytics?utm_source=oss&utm_media=static-embed-popover",
+      "https://www.metabase.com/product/embedded-analytics?utm_source=product&utm_medium=upsell&utm_campaign=embedding-interactive&utm_content=static-embed-popover&source_plan=oss",
```

Comment:
This is a test for
✅ Interactive embedding upsell
Overview doc
Look and Feel doc
Look and Feel font upsell